### PR TITLE
Use `eslint-env` instead of  disable `no-undef`

### PR DESCRIPTION
**Pull request by Hongarc**
Use `eslint-env` is better than `global` or disable `no-undef` when you use `mocha` or `jasmine`

I use: `//* eslint-env jasmine */` instead of `/* eslint-env jasmine */` to bypass `eslint-comments/no-use`

### CODE REVIEW

File: test/helpers/testing/jasmine/main.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
+//* eslint-env jasmine */
```
Comment (ehmicky) [inline test/helpers/testing/jasmine/main.js]:
```suggestion
/* eslint-env jasmine */
```

I've updated the `eslint-comments/no-use` configuration so `//*` is not needed anymore

File: test/helpers/testing/mocha/main.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
+//* eslint-env mocha */
```
Comment (ehmicky) [inline test/helpers/testing/mocha/main.js]:
```suggestion
/* eslint-env mocha */
```

File: test/helpers/testing/jasmine/main.js (hunk 1 above)
Comment (Hongarc) [reply test/helpers/testing/jasmine/main.js]: Ok, I will remove it, can you show me where update `eslint-comments/no-use`?

Comment (Hongarc) [reply test/helpers/testing/jasmine/main.js]: I see it, I update source but forgot re-install so I see the error.

**Pull request merged**